cypress/page/login.page.ts

```diff
@@ -0,0 +1,23 @@
+class LoginPage {
+
+    private email: string
+    private password: string
+    private signInBtn: string
+
+    constructor() {
+        this.email = "#email"
+        this.password = "#passwd"
+        this.signInBtn = "#SubmitLogin"
+    }
+
+    public typeUser(): void {
+        cy.get(this.email).type("aperdomobo@gmail.com")
```

Comment:
receive the email as param of the method

Reply:
Done